blog: October newsletter

GregSutcliffe opened the pull request:
As always, more content is welcome (although it's pretty big already!).

GregSutcliffe committed:
blog: October newsletter

GregSutcliffe commented:
[x] ~~Got a last minute addition of a Deep Dive to add, don't merge until I've done that.~~

Done

shlomizadok commented:
@GregSutcliffe++ that's an awesome write-up

ehelms commented on _posts/2016-10-27-foreman-community-newsletter---october-2016.md:
```diff
@@ -0,0 +1,180 @@
+[el6]: https://groups.google.com/d/topic/foreman-users/lec9OBPJEGo/discussion
+[userslist]: https://groups.google.com/forum/#!forum/foreman-users
+
+### Katello 3.2 RC2
```
RC3 is going out today with the EL6 support if you want to update this :)

GregSutcliffe replied:
This is unlikely to be merged today anyway, I'll update tomorrow morning with a link to the announce. Thanks!

ehelms replied:
Thanks!

GregSutcliffe replied:
Done!

ehelms commented:
You may want to throw in https://theforeman.org/2016/10/puppet4-integration.html to the blog section.

GregSutcliffe commented:
@ehelms Both added (the blog is questionable since it'll be on the same page and RSS as this newsletter... but I added it anyway :P). Also added @mclueppers blogs on HA too.

mmoll commented:
merged with small link fixes, thanks @GregSutcliffe!
